mongoose/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for Mongoose 4.7.0
 // Project: http://mongoosejs.com/
-// Definitions by: simonxca <https://github.com/simonxca/>, horiuchi <https://github.com/horiuchi/>
+// Definitions by: simonxca <https://github.com/simonxca/>, horiuchi <https://github.com/horiuchi/>, sindrenm <https://github.com/sindrenm>
```

Comment:
@sindrenm this means you'll be notified of future PRs and will help review them. If you're not interested in doing that, you should remove this.